src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Controllers/Domain/PlayersController.cs

```diff
@@ -322,16 +323,21 @@ private static List<int> ParseIntList(string source)
         {
             var splitted = source.Split(',');
             var result = new List<int>();
-            int parsed;
             foreach (var i in splitted)
             {
-                if (int.TryParse(i, out parsed))
+                if (int.TryParse(i, out var parsed))
                 {
                     result.Add(parsed);
                 }
             }
 
             return result;
         }
+
+        private bool IsFreePlayer(Player player, int? includeTeam)
```

Comment:
When you will refactor it?

Reply:
For which use case second parameter is needed?